tests/unit/generate_detect_target_contour.py

```diff
@@ -0,0 +1,183 @@
+"""
+Helper functions for `test_detect_target_contour.py`.
+
+"""
+
+import cv2
+import math
+import numpy as np
+
+
+LANDING_PAD_COLOR_BLUE = (100, 50, 50)
+
+
+# Test functions use test fixture signature names and access class privates
+# No enable
+# pylint: disable=protected-access,redefined-outer-name
+
+
+class LandingPadData:
+
+    def __init__(
+        self,
+        center: tuple[int, int],
+        axis: tuple[int, int],
+        blur: bool,
+        angle: int,
+    ):
+        """
+        Represents the data required to define and generate a landing pad.
+
+        Attributes:
+            center: The (x, y) coordinates representing the center of the landing pad.
+            axis: The lengths of the semi-major and semi-minor axes of the ellipse.
+            blur: Indicates whether the landing pad should have a blur effect. default: False.
+            angle (int): The rotation angle of the landing pad in degrees. defaults: 0.
```

Comment:
Remove type.

What values are allowed (e.g. is >360 allowed, is >180 allowed)? You can arbitrarily set more restrictive limits if you're lazy like me.

What direction is the semi major axis for an angle of 0? Which way does it rotate as the angle is increased?